### DISCUSSION TITLE
The ability to allocate virtual disk space

### DISCUSSION THREAD

**Discussion by walletje** [Genymobile/feedback#69] [Genymotion Desktop] [answered]
I'm missing the function to allocate more disk space to my virtual machines. It would be really nice to have more than what they give you to begin with.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @walletje,

We are sorry but this feature is not available with Genymotion Desktop at the moment.

However, if you are using Genymotion Desktop on Linux or macOS, there is a possible workaround to increase the virtuel disk space - see #160
